modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -3072,6 +3072,111 @@ export const console: ConsoleNS = {
                                     }
```

Comment:
It seems you have only added the `en` language. Please check and add to other languages as well.

Reply:
Tracking an issue for this. I will address them with https://github.com/wso2-enterprise/asgardeo-product/issues/5593